### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by mochenxin000** [emadeldeen24/AttnSleep#19]
Hi. I was just wondering, every time we practice. Is the split of the training set and the test set fixed? Why is it that the accuracy of my experiments is the same number of folds every time?

**Comment by emadeldeen24** [comment]
In the experiments, I used k-fold cross-validation. For each fold, I have train/test splits.
For example, if I have 20 subjects and running 2o-fold CV, the train set for fold 0 will be subjects 1->19, while the test will be subject 0. For fold 1, the test will be subject 1 and the others will be used for training, and so on.
